src/components/chat/comment-card.tsx

```diff
@@ -14,22 +15,27 @@ interface IState {
   commentAdded: boolean;
   commentText: string;
 }
+const minTextAreaHeight = 35;
 
 @inject("stores")
 @observer
 export class CommentCard extends BaseComponent<IProps, IState> {
+  state = {
+    commentTextAreaHeight: minTextAreaHeight,
+    commentAdded: false,
+    commentText: "",
+  };
 
-  constructor(props: IProps) {
-    super(props);
-    this.handleCommentTextAreaChange = this.handleCommentTextAreaChange.bind(this);
-    this.handleCancelPost = this.handleCancelPost.bind(this);
-    this.handleSendPost = this.handleSendPost.bind(this);
-
-    this.state = {
-      commentTextAreaHeight: 35,
-      commentAdded: false,
-      commentText: "",
-    };
+  escFunction = (event: any) => {
```

Comment:
It's conventional to have handler functions prefixed with `handle...`, e.g. `handleEscKey()` and props that refer to handler functions prefixed with `on...`, hence `onKeyDown={handleEscKey}` as recommended below.